data.sql

```diff
@@ -47,4 +47,29 @@ VALUES
 	    ('Angemon', '06-12-2005', 1, 'true', -45),
 	    ('Boarmon', '06-07-2005', 7, 'true', 20.4),
       ('Blossom', '10-13-1998', 3, 'true', 17),
-      ('Ditto', '05-14-2022', 4, 'true', 22);
\ No newline at end of file
+      ('Ditto', '05-14-2022', 4, 'true', 22);
+
```

Comment:
- [ ] Invalid date format. by default, Postgres uses the data format `year-month-day(Y-M-D)`, however  the date format it seems you are using is the format M-D-Y. Kindly add change this format to the Postgres default(Y-M-D). or add some interactions in your data.sql file to change the default Postgres date format.  this is the error I am getting :point_down: 

![image](https://user-images.githubusercontent.com/65068771/159387818-9543d0c2-26aa-4f40-b408-caffcb399a66.png)
![image](https://user-images.githubusercontent.com/65068771/159387890-a15581fe-2694-4589-91ac-36f03ec2e8e0.png)


here is how you can change postgress default date format :point_down:

```sql
/* Here is the commend "set datestyle to [your new datestyle];" */

/* Example: */
set datestyle to DMY;
```